src/lib/ui/components/File/File.svelte

```diff
@@ -38,22 +38,28 @@
   $: datasets = generateDatas(datas);
 
   function onSelectFile(event: Event) {
-    const { files: filesToUpload } = event.target as HTMLInputElement;
-    if (filesToUpload) {
-      const fileToUpload =
+    const { files: filesInput } = event.target as HTMLInputElement;
+    if (filesInput) {
+      const filesToUpload =
         multiple && files
```

Comment:
🚫 **[eslint]** <[operator-linebreak](https://eslint.org/docs/latest/rules/operator-linebreak)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>There should be no line break before or after '='.
```suggestion
      const filesToUpload =        multiple && files
```